spaces/domains.json

```
@@ -60,5 +60,6 @@
   "fund.emptyset.finance": "esd.eth",
   "mtrg.meter.io": "meter-mainnet.eth",
   "emtrg.meter.io": "meter-eth.eth",
-  "gov.kp2r.network": "keep2r.eth"
+  "gov.kp2r.network": "keep2r.eth",
+  "gov.metapool.finance": "metapool"
```

The domain selector should be similar to the space directory name.
```suggestion
  "gov.metapool.finance": "0xEaDC218Ac4cB7895a5A56e6484646b48f841C45a.eth"
```